<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калории и Белки</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>218</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Калькулятор Калорий и Белков</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCalories">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Калорийность на 100 г:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="caloriesInput">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelWeight">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Вес порции (г):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="weightInput">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelProteins">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Белки на 100 г:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="proteinsInput">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>360</width>
      <height>60</height>
     </rect>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
